doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -58,104 +70,115 @@ The following examples illustrate basic constituents of the configuration schema
+    "type":"record",
+    "fields":[
         {
-            "name": "stringField",
-            "type": "string",
-            "by_default": "default string value"
+            "name":"stringField",
+            "type":"string",
+            "by_default":"default string value"
         },
         {
-            "name": "intField",
-            "type": "int",
-            "by_default": 0
+            "name":"intField",
+            "type":"int",
+            "by_default":0
         }
     ]
 }
 ```
 
 The following table specifies the **by_default** attribute format for every supported primitive type.
 
-| Type     | Format                                   |              Example             |
-|----------|:----------------------------------------:|:--------------------------------:|
-| boolean  | true/false                               | "by_default": true               |
-| bytes    | json array of byte values                | "by_default": [1, 2, 55, 254, 4] |
-| double   | floating point value                     | "by_default": 1.432              |
-| float    | floating point value                     | "by_default": 1.432              |
-| int      | numeric value from (-2^31+1) to (2^31-1) | "by_default": 55                 |
-| long     | numeric value from (-2^63+1) to (2^63-1) | "by_default": 2147483648         |
-| string   | simple string format                     | "by_default": "abcdef"           |
-
-* The **addressable** record type field attribute (boolean, true by default) determines whether or not the record supports partial updates ([deltas]()). If this attribute is true, Kaa automatically adds a UUID field (\__uuid) to the record when producing derivative schemas, which is done for the addressing purposes. For this reason, \__uuid is a reserved field name in Kaa.  Note that the root record in the configuration schema is always addressable and thus ignores the statement _"addressable": false_.
+| Type           | Format                                   |              Example             |
+|----------------|:-----------------------------------------|:---------------------------------|
+| ```boolean```  | true/false                               | "by_default": true               |
+| ```bytes```    | json array of byte values                | "by_default": [1, 2, 55, 254, 4] |
+| ```double```   | floating point value                     | "by_default": 1.432              |
+| ```float```    | floating point value                     | "by_default": 1.432              |
+| ```int```      | numeric value from (-2^31+1) to (2^31-1) | "by_default": 55                 |
+| ```long```     | numeric value from (-2^63+1) to (2^63-1) | "by_default": 2147483648         |
+| ```string```   | simple string format                     | "by_default": "abcdef"           |
+
+* The **addressable** record type field attribute (```boolean```, ```true``` by default) determines whether or not the record supports partial updates
```

Comment:
please remove link to delta calculation algorithm. Add "the record supports partial updates from different endpoint groups"